proto/raft_serverpb.proto

```diff
@@ -117,10 +117,17 @@ enum ExtraMessageType {
     MsgHibernateRequest = 4;
     MsgHibernateResponse = 5;
     MsgRejectRaftLogCausedByMemoryUsage = 6;
+    MsgCollectPeerApplyIndexProgressRequest = 7;
+    MsgCollectPeerApplyIndexProgressResponse = 8;
```

Comment:
why need a response

Reply:
Oh, What I thought was that when the leader sends `MsgAppend` or `MsgHeartbeat`, it sends a `MsgCollectPeerApplyIndexProgressRequest`, and then other peers receive this information and return `MsgCollectPeerApplyIndexProgressResponse`.

But it should be that when non-leader peers send `MsgAppendResp` or `MsgHeartBeatResp`, they carry the `extra state` information, and when the leader receives `MsgAppendResp` or `MsgHeartBeatResp`, it processes the received information. So no need for an `extra resp type` right?

Reply:
Removed, thanks!